scripts/zones/Aht_Urhgan_Whitegate/npcs/Wondrix.lua

```diff
@@ -0,0 +1,40 @@
+-----------------------------------
+-- Area: Windurst Woods
+--  NPC: Arbitrix
```

Comment:
Wondrix is trying to steal Arbitrix's identity! Even falsely proclaiming their residence as Windurst Woods instead of Aht Urhgan Whitegate!